api/get_user_states.go

```diff
@@ -29,10 +30,8 @@ func (a *API) unsafeGetUserStates(ctx context.Context, publicKey *models.PublicK
 	span := trace.SpanFromContext(ctx)
 	span.SetAttributes(attribute.String("hubble.publicKey", publicKey.String()))
 
-	log.WithFields(o11y.TraceFields(ctx)).Infof("Getting leaves for public key: %s", publicKey.String())
```

Comment:
Is this remove intentional? (i.e. does it make the logs too noisy?)

Reply:
the remove is intentional, this line was spamming our logs. This is kind of bad form, it's an unrelated change, but I decided to change it while I was messing with this method